thunderbird/MozillaThunderbird.spec

```diff
@@ -56,10 +54,9 @@
 %define desktop_file_name %{progname}
 %define __provides_exclude ^lib.*\\.so.*$
 %define __requires_exclude ^(libmoz.*|liblgpllibs.*|libxul.*|libldap.*|libldif.*|libprldap.*)$
-
 %define localize 1
 %ifarch %ix86 x86_64
-%define crashreporter 1
+%define crashreporter 0
```

Comment:
providing crashreporter just for one arch is probably useless. Also since we currently do not have the possibility anymore to create and upload debug symbols the upstream crash collector cannot do useful things. So proposing to disable for now completely.

Reply:
Then we could remove this whole ifarch-section, because now we have "if x86, disable crashreporter, else disable crashreporter". Same on Firefox-side.

Reply:
doubled block is a merge artefact IMHO. And yes; I first tried to preserve the logic so it's not forgotten if we would like to enable crashreporter again at some point. Will update the branch